avaframe/in1Data/getInput.py

```diff
@@ -60,7 +60,8 @@ def getInputData(avaDir, cfg, flagDev=False):
     elif cfg['releaseScenario'] != '':
         releaseDir = 'REL'
         relFiles = []
-        releaseFiles = cfg['releaseScenario'].split('_')
+        releaseFiles = cfg['releaseScenario'].split('|')
```

Comment:
Please use the splitIniValueToArray function here...

Reply:
the splitIniValueToArray is not supposed to work for this case, as a string is required for the file name not a numpy array which is returned in splitIniValueToArray, I could make an additional function but so far this is the only case where we need this and it is one line so I'd suggest to keep it here